### PR TITLE
Fixed validateExpression property not returned in sling model

### PR DESCRIPTION
**Pull request by ravise5**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java [hunk 1]
```diff
@@ -451,7 +448,15 @@ public Type getType() {
 
     @Override
     @Nullable
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     public String getValidationExpression() {
-        return validationExpression;
+        Resource fdRules = resource.getChild("fd:rules");
```
Comment (nit23uec) [inline bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java:453]: minor: pls use constant: `CUSTOM_RULE_PROPERTY_WRAPPER`

File: bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/AbstractContainerImplTest.java [hunk 2]
```diff
@@ -49,4 +49,11 @@ void testGetDorProperties() {
         assertEquals(null, abstractContainerMock.getDorProperties().get("dorTemplateRef"));
     }
 
+    @Test
+    void testGetValidationExpression() {
+        AbstractContainerImpl abstractContainerMock = Mockito.mock(AbstractContainerImpl.class);
+        Mockito.when(abstractContainerMock.getDorProperties()).thenCallRealMethod();
+        assertEquals(null, abstractContainerMock.getValidationExpression());
```
Comment (nit23uec) [inline bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/AbstractContainerImplTest.java:56]: lets have an actual test case where validationExpression is not null in jcr.

Comment (ravise5) [reply bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/AbstractContainerImplTest.java:56]: sure

Comment (ravise5) [reply bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/AbstractContainerImplTest.java:56]: @nit23uec I suppose that would be an IT test case..

**Comment by vdua** [comment]
closing this. Will fix that in rule editor since we can do that there.

**Pull request closed without merging**